test_stbt_rig.py

```diff
@@ -226,6 +227,13 @@ def test_run_tests_interactive(capsys, test_pack, tmpdir, portal_mock):
         'stbt_rig.py', '--node-id=mynode', '--portal-url=%s' % portal_mock.url,
         '--portal-auth-file=token', 'run', 'tests/test.py::test_my_tests'])
```

Comment:
Is there any difference between this second invocation of `stbt_rig.py` and the first invocation?

Reply:
Looks like a copy-paste error, removed.